product_categ_image/views/product_category_views.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="product_category_form_view" model="ir.ui.view">
+        <field name="name">product.category.image.form</field>
+        <field name="model">product.category</field>
+        <field name="inherit_id" ref="product.product_category_form_view" />
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='name']/../.." position="before">
+                <field name="image_1920" widget="image" class="oe_avatar oe_left" />
```

Comment:
If you do something like this:
                <group>
                    <div>
                        <field name="image_1920" widget="image" class="oe_avatar oe_left"/>
                    </div>
                 </group>
The Category name label won't be next to the image. I think that looks a bit nicer.